packages/core/src/presets/presetManager.ts

```
@@ -0,0 +1,101 @@
+        return OrionPreset;
+      case 'streamfusion':
+        return StreamFusionPreset;
+      case 'anime-kitsu':
+        return AnimeKitsuPreset;
+      case 'nuvio-streams':
+        return NuvioStreamsPreset;
+      default:
+        throw new Error(`Preset ${id} not found`);
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)